buf.gen.yaml

```diff
@@ -1,12 +1,15 @@
-version: v1
+version: v2
 managed:
   enabled: true
-  go_package_prefix:
-    default: github.com/bufbuild/protovalidate-go/internal/gen
-    except:
-      - buf.build/envoyproxy/protoc-gen-validate
-      - buf.build/bufbuild/protovalidate
+  disable:
+    - file_option: go_package
+      module: buf.build/envoyproxy/protoc-gen-validate
+    - file_option: go_package
+      module: buf.build/bufbuild/protovalidate
+  override:
+    - file_option: go_package_prefix
+      value: github.com/bufbuild/protovalidate-go/internal/gen
 plugins:
-  - plugin: buf.build/protocolbuffers/go:v1.32.0
+  - remote: buf.build/protocolbuffers/go:v1.32.0
```

Comment:
Do you want to bump this to a newer version?

Reply:
Suppose I could since I'm in here